### DISCUSSION TITLE
Question regarding phase unwrapping

### DISCUSSION THREAD

**Discussion by loneblues** [kschan0214/sepia#71] [Q&A]
Hello. 
For phase unwrapping, there is option for mask. 
I want to ask if the brain mask provided (which is better than default BET), can be used for such purposes. 
Thanks. 

Best, 
Ho-Joon Lee

**Comment by kschan0214** [comment]
Hi Ho-Joon,

Yes, you can specify your own mask on the GUI:
<img width="1120" alt="Screenshot 2023-11-01 at 22 23 07" src="https://github.com/kschan0214/sepia/assets/39571502/7d8bbb5b-2931-4186-8c24-3e9ac7e1dcf7">

The mask file has to be in the NIFTI format (.nii or .nii.gz).

Let me know if you encounter any issue.

Best,

Kwok
